aspnetcore/signalr/security.md

```diff
@@ -105,16 +105,20 @@ In ASP.NET Core 2.1 and later, header validation can be achieved using a custom
 
 ::: moniker-end
 
+## ConnectionID
```

Comment:
@wadepickett  fix this all all the others. Do a global change.

Reply:
I was finally able to see the option for committing for a suggestion or add to batch with halter73's change.  However your suggested change is still grayed out for committing or add to batch.  Very odd.  So I could not batch both of them.  To see if commit really worked I went ahead and committed halter73's suggestion and that worked.  Very odd behavior but another clue to what may be going on.

Reply:
@Rick-Anderson , could you try to add one more change, on the date or something?  Maybe it commiting/batch will work going forward for future changes, but just will not for the past one you made while I didn't have expanded rights?

Reply:
..and I will fix the others as well.  I just got excited to see the commit feature working...

Reply:
ConnectionId fixed everywhere.  Thanks.